src/shared/tpm2-util.h

```diff
@@ -67,9 +73,14 @@ Tpm2Context *tpm2_context_unref(Tpm2Context *context);
 DEFINE_TRIVIAL_CLEANUP_FUNC(Tpm2Context*, tpm2_context_unref);
 #define _cleanup_tpm2_context_ _cleanup_(tpm2_context_unrefp)
 
+typedef enum TPM2HandleFlags {
+        TPM2_HANDLE_FLAGS_NOFLUSH = 1 << 0,
```

Comment:
doesn't matter much, but usually it is nicer to express things "positively" instead of "negatively". hence maybe rename this "keep" or something, rather than "noflush"?

Reply:
I can change it, I was following the dm-ioctl.h DM_NOFLUSH_FLAG.

Reply:
oh that's a public kernel api. they need their stuff to be extensible without breaking structure arrangements. that doesn't apply to an internal structure in our codebase.

i mean, flags are fine, and maybe we want this to be a flag one day if we have so many of them. but as long as it is just one or two i'd just make this a bool (and once it is two a bitfield, i.e. `bool x:1;`

Reply:
> oh that's not the best model, it's a public kernel api. they aren't great with naming and they need their stuff to be extensible without breaking structure arrangements. that doesn't apply to an internal structure in our codebase.
> 

I just hate updating code bases where a bool goes to flags and theirs like 300 spots. I guess since this is internal to tpm2 it's unlikely to happen.

> i mean, flags are fine, and maybe we want this to be a flag one day if we have so many of them. but as long as it is just one or two i'd just make this a bool (and once it is two a bitfield, i.e. `bool x:1;`

Sure, bool it is.